compose/install.md

```diff
@@ -34,28 +34,59 @@ To install Compose, do the following:
     Invoke-WebRequest "https://github.com/docker/compose/releases/download/$dockerComposeVersion/docker-compose-Windows-x86_64.exe" -UseBasicParsing -OutFile $Env:ProgramFiles\docker\docker-compose.exe
     ```
 
-    For example, to download Compose version 1.12.0, the command is:
+    For example, to download Compose version 1.13.0, the command is:
 
     ```none
-    Invoke-WebRequest "https://github.com/docker/compose/releases/download/1.12.0/docker-compose-Windows-x86_64.exe" -UseBasicParsing -OutFile $Env:ProgramFiles\docker\docker-compose.exe
-        ```
+    Invoke-WebRequest "https://github.com/docker/compose/releases/download/1.13.0/docker-compose-Windows-x86_64.exe" -UseBasicParsing -OutFile $Env:ProgramFiles\docker\docker-compose.exe
+    ```
+    >**Use the latest Compose release number in the download command.**
+    As already mentioned, the above command is an _example_, and it may become
+out-of-date once in a while. Always follow the command pattern shown above it.
+If you cut-and-paste an example, check which release it specifies and, if
+needed, replace `$dockerComposeVersion` with the release number that you want.
+Compose releases are also available for direct download on the
+[Compose repository release page on
+GitHub](https://github.com/docker/compose/releases){: target="_blank"
+class="_"}.
+    {: .warning-notitle}
 
     Now, run the executable to install Compose.
 
 3.  On **Linux**, you can download the Docker Compose binary from the
     [Compose repository release page on GitHub](https://github.com/docker/compose/releases){: target="_blank" class="_"}.
     Follow the instructions from the link, which involve running the `curl` command in your terminal to download the binaries.
 
-    > **Note**: If you get a "Permission denied" error, your `/usr/local/bin` directory
-    > probably isn't writable and you'll need to install Compose as the superuser. Run
-    > `sudo -i`, then the two commands below, then `exit`.
+    > **Did you get a "Permission denied" error?** If so, your
+    `/usr/local/bin` directory probably isn't writable and you'll
+    need to install Compose as the superuser. Run `sudo -i`, then run the
+    download and install commands below, then `exit`.
+    {: .note-notitle}
+
 
-    The following is an example command illustrating the format:
+    Run this command to download Docker Compose, replacing
+`$dockerComposeVersion` with the specific version of Compose you want to use:
 
     ```bash
-    curl -L https://github.com/docker/compose/releases/download/1.12.0/docker-compose-`uname -s`-`uname -m` > /usr/local/bin/docker-compose
+    curl -L https://github.com/docker/compose/releases/download/$dockerComposeVersion/docker-compose-`uname -s`-`uname -m` > /usr/local/bin/docker-compose
```

Comment:
I am using the same commands as on [Compose releases page](https://github.com/docker/compose/releases). I've had this comment before, did testing, and decided to match what is on the releases page.